jest.process.env.sample.js

```diff
@@ -0,0 +1,4 @@
+process.env.AWS_SMOKE_TEST_BUCKET = "";
+process.env.AWS_SMOKE_TEST_REGION = "";
+
+// copy this file to jest.process.env.js
```

Comment:
This is no longer needed, as tests will be run from `test/e2e`

You may add README like https://github.com/aws/aws-sdk-js-v3/blob/main/clients/client-s3/test/e2e/README.md, but since the test code lies in src folder, a comment at the top of the test file should be enough

> // Run using AWS_SMOKE_TEST_REGION=[Region] AWS_SMOKE_TEST_BUCKET=[Bucket] yarn test:e2e